examples/ota-requestor-app/esp32/main/main.cpp

```diff
@@ -73,6 +74,35 @@ static void InitServer(intptr_t context)
 
 } // namespace
 
+// Test mode operation
+void NotifyUpdateAppliedHandler(Layer * systemLayer, void * appState)
+{
+    OTARequestor * requestorCore = static_cast<OTARequestor *>(appState);
+    uint32_t softwareVersion;
+    // TODO : With persistent storage working later on in the OTA Requestor core,thisnd the calling of TestModeSetProviderParameters
+    // will not be needed later.
+    NodeId nodeId;
+    FabricIndex fabIndex;
+    EndpointId endpointId;
+    esp_err_t err = OTARequestorNVSHelper::ReadProviderLocation(nodeId, fabIndex, endpointId);
+
+    if (err != ESP_OK)
+    {
+        ESP_LOGE(TAG, "Failed to read provider location: %s", esp_err_to_name(err));
+        return;
+    }
+    requestorCore->TestModeSetProviderParameters(nodeId, fabIndex, endpointId);
```

Comment:
FYI, my [PR](https://github.com/project-chip/connectedhomeip/pull/15170) merged which removes this API. To test moving forward, you should set the DefaultOTAProviders attribute instead of explicitly setting from the app.